keps/sig-instrumentation/1668-trace-popagating/README.md

```diff
@@ -0,0 +1,501 @@
+# KEP-1668: Trace popagating
```

Comment:
nit: Can we name this Trace Context Propagation?  "Context" is the important word here.

Reply:
(please also rename the file - there is a typo)

Reply:
Thanks, i will update it soon.